### DISCUSSION TITLE
Tracking the mass of water within a volume

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#13024] [General]
I am attempting to track the mass of water within a given volume, but it seems that FDS is only capable of tracking the mass that contacts a surface/obstruction. I have read throughout the FDS user manual and verification .fds files in the repository but can't seem to find anything that does what I am looking for. I would greatly appreciate if anyone is able to confirm whether or not FDS is capable of tracking mass within a volume that does not contact an obstruction or surface. Thanks!

**Comment by drjfloyd** [comment]
22.9.3 Droplet and Particle Densities and Fluxes in the Gas Phase

Away from solid surfaces, ’MPUV’ is the Mass Per Unit Volume of particles or droplets of type (PART_ID)
in units of kg/m3. The averagae volume fraction of droplets in the cell is ’DROPLET VOLUME FRACTION’,
which is equal to the MPUV divided by the liquid density of the droplets. ’MPUV_Z’ provides the same
information integrated over all droplets of a single species, (SPEC_ID).
